lib/inferno/entities/ig.rb

```diff
@@ -15,6 +15,7 @@ class IG < Entity
       ATTRIBUTES = [
         :id,
         :resources_by_type,
+        :value_sets,
```

Comment:
This shouldn't be necessary since value_sets is now defined explicitly below

Reply:
Removed https://github.com/inferno-framework/inferno-core/pull/636/commits/25bf8422ec24b23fa280c8147d0b135257c29c09